circuit-construction/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
```

Comment:
Where does it come from?

Reply:
I believe this is something @mrmr1993 added when he was working on an earlier version of this PR. This existed in the previous PR when I picked up the task :shrug: